### PR TITLE
updated docstring format

### PR DESCRIPTION
**Pull request by andrewhong04**
I saw that some of the older files had some docstring formats that weren't consistent with the newer files. I've updated some of them, but I didn't update the ones that I either missed or some information in the docstring was missing.

### CODE REVIEW

File: scripts/tuxepedia/api.py [hunk 1]
```diff
@@ -83,11 +83,14 @@ def sync_with_remote(self, completed_monsters):
                     json.dump(txmn_json_full[txmn_name], f, indent=4)
 
     def update_txmn_json(self, txmn_name, txmn_json_new, overwrite=True):
-        """Update a tuxemon JSON file record
+        """
+        Update a tuxemon JSON file record
+
+        Parameters:
+            txmn_name: tuxemon name
```
Comment (vnmabus) [inline scripts/tuxepedia/api.py]: Start descriptions by uppercase and end them with a period.

File: scripts/tuxepedia/extractor.py [hunk 2]
```diff
@@ -39,9 +39,11 @@ def get_logger(self):
         return logging.getLogger(self.__class__.__name__)
 
     def get_completed_monsters(self):
-        """Extract monster data from the Tuxepedia Wiki page
+        """
+        Extract monster data from the Tuxepedia Wiki page.
 
-        :return: dict/JSON of the Tuxepedia monster entries
+        Returns:
+            dict/JSON of the Tuxepedia monster entries
```
Comment (bitcraft) [inline scripts/tuxepedia/extractor.py]: @vnmabus whats your opinion on this?  should we treat the returns like everything else?  Capital letters and periods?

Comment (vnmabus) [reply scripts/tuxepedia/extractor.py]: I think so, yes: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html .

**Review by bitcraft — APPROVED** [review]
Great updates here, thank you!

**Comment by bitcraft** [comment]
@novialriptide The changes are good, but there is a merge conflict that needs to be resolved before I can merge it.

**Pull request merged**